Aztec/Classes/Libxml2/DOMString.swift

```diff
@@ -493,6 +493,23 @@ extension Libxml2 {
             rootNode.replaceCharacters(inRange: range, withElement: elementDescriptor)
         }
 
+        /// Applies horizontal ruler to the specified range.
+        ///
+        /// - Parameters:
+        ///     - range: the range to apply the style to.
+        ///
+        func insertHorizontalRuler(at range: NSRange) {
```

Comment:
Can we change the name of this method to better depict that it's replacing existing contents?

Insert makes it sound as if this method didn't affect existing content.

`replaceWithHorizontalRuler(_ range: NSRange)` maybe?